src/main/java/com/gmail/inverseconduit/javadoc/ClassInfoXmlParser.java

```diff
@@ -0,0 +1,179 @@
+package com.gmail.inverseconduit.javadoc;
+
+import java.util.Arrays;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.gmail.inverseconduit.utils.DocumentWrapper;
+
+/**
+ * Parses {@link ClassInfo} objects from XML documents.
+ * @author Michael Angstadt
+ */
+public class ClassInfoXmlParser {
```

Comment:
Concerning the "Javadocs"... the comments inside the methods are not javadocs. they're comments. they should explain the **why** and not the **what** - the code is responsible for explaining the what. Consider extracting such things to methods. 

The method name gives you a place to document what happens ;)